### PR TITLE
Add signer() and verifier() returning functions for use with jsigs

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #6, for use with https://github.com/veres-one/did-veres-one/issues/10, once https://github.com/digitalbazaar/jsonld-signatures/pull/45 is completed.

### CODE REVIEW

File: lib/ld-key-pair.js [hunk 1]
```diff
@@ -28,6 +30,7 @@ class LDKeyPair {
    */
   constructor(options) {
     this.injector = options.injector || new Injector();
+    this.algorithm = options.algorithm;
```
Comment (dlongley) [inline lib/ld-key-pair.js]: I don't think we have a need for `algorithm`, do we?

Comment (dmitrizagidulin) [reply lib/ld-key-pair.js]: Agreed, we don't need it now.

**Comment by dmitrizagidulin** [comment]
Algorithm removed, changes from https://github.com/digitalbazaar/jsonld-signatures/pull/45 extracted and integrated. Ready for review.

File: lib/ld-key-pair.js [hunk 2]
```diff
@@ -352,6 +354,34 @@ class RSAKeyPair extends LDKeyPair {
...
+    if(this.publicKeyPem) {
+      const publicKey = forge.pki.publicKeyFromPem(this.publicKeyPem);
+      const keyBits = publicKey.n.bitLength();
+      if(keyBits !== DEFAULT_RSA_KEY_BITS) {
+        throw new Error(`Invalid RSA keyBit length ${JSON.stringify(keyBits)}` +
+          ` required value is ${DEFAULT_RSA_KEY_BITS}`);
+      }
+      if(publicKey.e.toString(16) !== '10001') {
+        throw new Error(`Invalid RSA exponent ${JSON.stringify(publicKey.e.toString(16))}` +
+          ` required value is 10001}`);
```
Comment (dlongley) [inline lib/ld-key-pair.js]: Should probably say the value is 65537 here since 10001 is in hex and may confuse readers.

Comment (dlongley) [reply lib/ld-key-pair.js]: Could to `toString(10)` and compare against that for clarity.

Comment (dmitrizagidulin) [reply lib/ld-key-pair.js]: Ah yeah, good call. Changed.

**Comment by dmitrizagidulin** [comment]
@dlongley should we wait, tho? That seems like a good candidate for another PR (since the API won't change, it's just a refactoring)

**Comment by dlongley** [comment]
@dmitrizagidulin, ok, don't wait, proceed! :)

**Pull request merged**